views/eApostilles/startPage.ejs

```diff
@@ -0,0 +1,31 @@
+<div class="inner_header no-user-signed-in">
```

Comment:
When you hit this page, you loose the 'logged in' breadcrumb links for Applications, Account, Addresses and Sign out